Tools/LyTestTools/tests/unit/test_o3de_material_editor_test.py

```diff
@@ -0,0 +1,64 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+import unittest
+
+import pytest
+import unittest.mock as mock
+
+import ly_test_tools.o3de.material_editor_test as material_editor_test
+from ly_test_tools.launchers import launcher_helper
+
+pytestmark = pytest.mark.SUITE_smoke
+
+
+class TestMaterialEditorTestBase(unittest.TestCase):
+
+    def test_MaterialEditorSharedTest_Init_CorrectAttributes(self):
+        mock_materialeditorsharedtest = material_editor_test.MaterialEditorSharedTest()
+        assert mock_materialeditorsharedtest.timeout == 180
+        assert mock_materialeditorsharedtest.test_module is None
+        assert mock_materialeditorsharedtest.attach_debugger is False
+        assert mock_materialeditorsharedtest.wait_for_debugger is False
+        assert mock_materialeditorsharedtest.is_batchable
+        assert mock_materialeditorsharedtest.is_parallelizable
```

Comment:
Ditto, I don't think these tests provide much value beyond annoying someone who changes e.g. the default timeout.